tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/recommendations-validator.js

```diff
@@ -0,0 +1,12 @@
+import { selfAssessmentPageNewRecommendation } from "./completion-tags-validator.js";
+
+/**
+ * Validates a recommendation for a specific maturity level.
+ *
+ * @param {Object} section - the section containing the recommendations
+ * @param {string} maturity - the maturity level to validate
+ */
+
+export const validateRecommendationForMaturity = (section, maturity) => {
+    selfAssessmentPageNewRecommendation(section, maturity);
+}
```

Comment:
Since this just mirrors `completion-tags-validator` would it make sense to rename that one to `recommendations-validator` and remove this?

Reply:
Still adding to this fn